isis/src/base/objs/ImageOverlapSet/ImageOverlapSet.cpp

```diff
@@ -444,48 +444,46 @@ namespace Isis {
    * @param filename The file to write the image overlaps to
    */
   void ImageOverlapSet::WriteImageOverlaps(const QString &filename) {
-            
+
     QString file = FileName(filename).expanded();
     bool failed = false;
     bool noOverlaps = false;
-    
+
     if (p_threadedCalculate) {
       p_calculatePolygonMutex.lock();
     }
-    
-    try {
-      // Let's get an ostream pointed at our file
-      std::ofstream outStream;
 
+    // Let's get an ostream pointed at our file
+    std::ofstream outStream;
+
+    try {
       QByteArray fileArray = file.toLatin1();
       if (p_writtenSoFar == 0) {
         outStream.open(fileArray.constData(), fstream::out | fstream::trunc | fstream::binary);
       }
       else {
         outStream.open(fileArray.constData(), fstream::out | fstream::app | fstream::binary);
       }
-      
+
       failed |= outStream.fail();
 
-      static bool overlapWritten = false;
       for (int overlap = p_writtenSoFar; !failed && overlap <= p_calculatedSoFar; overlap++) {
         // Let's not try anything during a possible reallocate
         p_lonLatOverlapsMutex.lock();
-        
+
         if (p_lonLatOverlaps.size() == 0) {
           noOverlaps = true;
         }
         else {
           if (overlap < p_lonLatOverlaps.size() && p_lonLatOverlaps[overlap]) {
-                   
+
             if (!p_lonLatOverlaps[overlap]->Polygon()->isEmpty()) {
-                        
-              if (overlapWritten) {
+
+              if (p_writtenSoFar) {
```

Comment:
Yes but this would never have come up with how findimageoverlaps works as an application. OverlapWritten was a static bool and maintaining its truthiness between executions of the findimageoverlaps function